src/style/style.scss

```diff
@@ -1,17 +1,37 @@
 @import "globals";
 
+
+
+
+
+$default-font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Helvetica Neue", "Ubuntu", sans-serif;
+$header-font-family: 'Varela Round', sans-serif;
+$canvas-font-family: sans-serif;
```

Comment:
Not sure what's different for canvas vs default? 
Perhaps they should be combined?